src/gpu/intel/jit/v2/conv/kernel_desc.hpp

```diff
@@ -77,18 +77,21 @@ GPU_DEFINE_PARSE_ENUM(specialization_mode_t, specialization_mode_names)
 
 struct specialization_t {
     specialization_mode_t mode;
-    // Dimesion values to specialize.
-    pvar_tile_t spec_tile;
+    // Dimesion values to specialize (e.g. kw1).
```

Comment:
[nitpick]
```suggestion
    // Dimension values to specialize (e.g. kw1).
```
Here and below.

Reply:
Thanks, fixed.